.github/workflows/ci-base-tests.yml

```diff
@@ -1,10 +1,11 @@
 name: SMARTS CI Base Tests
 
-on: [push]
+on: [push, pull_request]
 
 jobs:
   test:
     runs-on: ubuntu-18.04
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
```

Comment:
This is a funny looking solution for the doubled up tests.

Reply:
Unfortunately, GitHub Actions does not provide a method to stop duplicated actions. Hence, the next best method would be to include a conditional statement. 

This conditional statement ensures that this test will only be triggered by
+ a `pull_request` that originates from a fork, or
+ a `push`.